actors/reward/src/types.rs

```diff
@@ -5,6 +5,20 @@ use fvm_ipld_encoding::tuple::*;
 use fvm_shared::address::Address;
 use fvm_shared::econ::TokenAmount;
 
+// #[derive(Debug, PartialEq, Eq, Clone, Default)]
```

Comment:
Commented code?

Reply:
Removed, thanks -- was related to [this](https://github.com/filecoin-project/builtin-actors/pull/913#discussion_r1049970729), but this isn't the PR for these changes.